csfieldguide/templates/base.html

```diff
@@ -18,99 +18,82 @@
     <link rel="stylesheet" href="{% static 'css/website.css' %}">
     <link rel="stylesheet" href="{% static 'css/third-party/featherlight.min.css' %}">
     <link rel="shortcut icon" href="{% static 'img/guide-favicon.png' %}">
-    <script src="{% static 'js/third-party/modernizr.js' %}"></script>
+    <script type="text/javascript" src="{% static 'js/third-party/modernizr.js' %}"></script>
+    <script type="text/javascript" src="{% static 'js/third-party/jquery.js' %}"></script>
```

Comment:
jQuery should be loaded at the end of the page, not within `<head>`. There is some great explanations why here: https://stackoverflow.com/questions/2105327/should-jquery-code-go-in-header-or-footer

Reply:
yes I agree, but it needs to be loaded first in order for inline interactives to work. This seems terrible and like something that can be fixed, but not in a one line change that I know off the top of my head :(